src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java

```diff
@@ -59,8 +59,9 @@ public BioHanselSampleUpdater(MetadataTemplateService metadataTemplateService, S
 	 * <p>
 	 * Create a bio_hansel {@link MetadataTemplate} for each {@link Project} of the {@link Sample} if one doesn't exist.
 	 *
-	 * @param samples  The sample to update (collection should only have one {@link Sample} object).
-	 * @param analysis Use the results from this {@link AnalysisSubmission} to update the {@link Sample} metadata.
+	 * @param  samples                 The sample to update (collection should only have one {@link Sample} object).
+	 * @param  analysis                Use the results from this {@link AnalysisSubmission} to update the {@link Sample}
+	 *                                 metadata.
```

Comment:
With the intellij formatter the metadata is on the same line as the @param above

Reply:
Fixed in ca919b6